examples/3d/lightmaps.rs

```diff
@@ -8,7 +8,7 @@ fn main() {
         .add_plugins(DefaultPlugins)
         .insert_resource(AmbientLight {
             color: Color::WHITE,
-            brightness: 0.2,
+            brightness: 0.0,
```

Comment:
Is there a (non-zero) default value for the ambient light? Because it doesn't seem like inserting a light with brightness set to zero should actually do much of anything

Reply:
Yes, the default is 0.005 or something. If you don't want any ambient light (using lightmaps or IBL instead), then you need to override it to be 0 like this.

Reply:
Yes it has a non-zero default. https://github.com/bevyengine/bevy/blob/main/crates/bevy_pbr/src/light.rs#L550